pkg/kev/converter/kubernetes/transform_test.go

```diff
@@ -793,13 +803,20 @@ var _ = Describe("Transform", func() {
 		})
 
 		When("project service label instructing to expose the k8s service is specified as `domain.name`", func() {
+			ingressAnnotations := map[string]string{
```

Comment:
I guess we could omit annotations in this particular block as we primarily test that domain(s) is set correctly?

Reply:
We need the ingress annotations to ensure we create a correct `&networkingv1beta1.Ingress` object.

Reply:
Sure, but I since this particular context block focuses on asserting correctness of domain name(s) I thought removing annotations will make it easier to read and follow. In any case ingress annotations are optional and you should be getting correct `Ingress` object even if they aren't specified. I believe there is another context there testing for annotations injection.